src/EventStore.ClusterNode.Web/styles/app.css

```
@@ -2,6 +2,7 @@
 @import "tailwindcss/utilities.css" layer(utilities) source(none);
 
 @source "../clusternode-web/index.html";
+@source "../../EventStore.ClusterNode/Components/**/*.razor";
```

Stylelint (17.9.0)

[error] Unexpected unknown at-rule "@source" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)